lib/ii.c

```diff
@@ -201,11 +223,14 @@ uint8_t* ii_processLeadRx( void )
 static void lead_callback( uint8_t address, uint8_t command, uint8_t* rx_data )
 {
     ii_unpickle( &address, &command, rx_data );
```

Comment:
Unfortunately we can't avoid this `unpickle` function unless there was a 'is_raw' state variable added to the ii_q_t type which was set globally in ii_leader_process (to be checked here).

It's probably a rare enough use case that we needn't worry. Perhaps this means the pickle function should be left as is above (though it does allow more flexibility when only *sending* data).